Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -5,121 +5,120 @@
 
   <info name="FCCDectDimensions"
     title="master file with includes and world dimension"
-    author="C. Helsens and J. Lingemann"
+    author="C. Helsens, J. Lingemann"
     url="no"
     status="development"
     version="$Id:??$">
```

Comment:
are we going to start using this field at some point? Maybe now is a good moment?